modules/test/playwright/pages/product-navigation-control-menu-web/ProductMenuPage.ts

```diff
@@ -99,9 +99,16 @@ export class ProductMenuPage {
 		await this.membershipsButton.click();
 	}
 
-	async goToPages() {
-		await this.siteBuilderButton.click();
-		await this.pagesButton.click();
+	async goToPages(siteUrl?: string) {
+		if (siteUrl) {
+			await this.page.goto(
+				`/group${siteUrl || '/guest'}${PORTLET_URLS.pages}`
+			);
+		}
+		else {
```

Comment:
@ethib137 we need the old method due to failures in `layout-set-prototype-web`. See https://github.com/liferay-platform-experience/liferay-portal/pull/776#issuecomment-2452647095

Reply:
It shouldn't break anything if we also update their usages to pass in the siteURL. I'm guessing it broke because they are using `isolatedSiteTest` which means it will create a new site for the test. This means if you don't pass in the siteURL it would go to a different site. It's understandable that would break it.

Reply:
@ethib137 the `layout-set-prototype-web` is creating a new site based on a site template. The url that gets output is incorrect when we pass `/group/template-the-id` to siteURL. It outputs

`http://localhost:8080/group/group/template-37476/~/control_panel/manage?p_p_id=com_liferay_layout_admin_web_portlet_GroupPagesPortlet`

when it should be

`http://localhost:8080/group/template-37476/~/control_panel/manage?p_p_id=com_liferay_layout_admin_web_portlet_GroupPagesPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view&p_r_p_selPlid=0&_com_liferay_layout_admin_web_portlet_GroupPagesPortlet_mvcRenderCommandName=%2Flayout_admin%2Fselect_layout_page_template_entry&_com_liferay_layout_admin_web_portlet_GroupPagesPortlet_redirect=http%3A%2F%2Flocalhost%3A8080%2Fgroup%2Ftemplate-37476%2F%7E%2Fcontrol_panel%2Fmanage%3Fp_p_id%3Dcom_liferay_layout_admin_web_portlet_GroupPagesPortlet%26p_p_lifecycle%3D0%26p_p_state%3Dmaximized%26p_p_auth%3DHygqpzR0&_com_liferay_layout_admin_web_portlet_GroupPagesPortlet_backURL=http%3A%2F%2Flocalhost%3A8080%2Fgroup%2Ftemplate-37476%2F%7E%2Fcontrol_panel%2Fmanage%3Fp_p_id%3Dcom_liferay_layout_admin_web_portlet_GroupPagesPortlet%26p_p_lifecycle%3D0%26p_p_state%3Dmaximized%26p_p_auth%3DHygqpzR0&_com_liferay_layout_admin_web_portlet_GroupPagesPortlet_groupId=37477&_com_liferay_layout_admin_web_portlet_GroupPagesPortlet_privateLayout=true&_com_liferay_layout_admin_web_portlet_GroupPagesPortlet_selectedTab=basic-templates&p_p_auth=HygqpzR0`

Do you have any idea how I could get that url?

Reply:
Hey @pat270 you don't need that whole url... the problem with `http://localhost:8080/group/group/template-37476/~/control_panel/manage?p_p_id=com_liferay_layout_admin_web_portlet_GroupPagesPortlet` is that it includes `group/group` when it should just be `group`. If you only have one group then that url works.

Reply:
ooh nice catch, I was pulling my hair out over this.